### PR TITLE
cleanup dead code, rename js folder

### PR DESCRIPTION
**Pull request by btry**
scripts/combobox.js.php  is not used anywhere
scripts/forms-validation.js.php has one function always returning true

rename scripts folder into js

### CODE REVIEW

File: scripts/forms-validation.js.php [hunk 1]
```diff
@@ -1,18 +0,0 @@
-<?php
-include ('../../../inc/includes.php');
-header('Content-type: text/javascript');
-?>
-
-function validateForm(form) {
```
Comment (orthagh) [inline scripts/forms-validation.js.php]: are you sure about validateForm removal? 

cf: https://github.com/pluginsGLPI/formcreator/blob/master/inc/form.class.php#L838

File: scripts/combobox.js.php [hunk 2]
```diff
@@ -1,136 +0,0 @@
-<?php
```
Comment (orthagh) [inline scripts/combobox.js.php]: for this part, we have some leftovers in css/style.css

File: scripts/forms-validation.js.php (hunk 1 above)
Comment (btry) [inline scripts/forms-validation.js.php]: You're right, I cancel removal of this file

Comment (btry) [inline scripts/forms-validation.js.php]: yes : the content of the function is commented out to always return true

File: scripts/combobox.js.php (hunk 2 above)
Comment (btry) [inline scripts/combobox.js.php]: I removed a few CSS based on the removed code, which I don't find anywhere else in the plugin.

**Pull request merged**